CHANGELOG.md

```diff
@@ -118,3 +118,5 @@ significant modifications will be credited to OpenTelemetry Authors.
  [#432](https://github.com/open-telemetry/opentelemetry-demo/pull/432)
 * Replaced the Jaeger exporter to the OTLP exporter in the OTel Collector
 ([#435](https://github.com/open-telemetry/opentelemetry-demo/pull/435))
+* Added cache scenario to recommendation service
+([#455](https://github.com/open-telemetry/opentelemetry-demo/pull/455))
```

Comment:
```suggestion
([#455](https://github.com/open-telemetry/opentelemetry-demo/pull/455))

```

Reply:
This is the same line?

Reply:
We need a blank like before EOF

Reply:
oh, right, doh. the web editor won't commit that because it doesn't see the diff. fixed.